result/accessor-pairs.js

```diff
@@ -0,0 +1,99 @@
+// Bad
+var o = {
+    set a(value) {
+        this.val = value;
+    }
+};
+
+// Good
+var o = {
+    set a(value) {
+        this.val = value;
+    },
+    get a() {
+        return this.val;
+    }
+};
+
+/*eslint accessor-pairs: "error"*/
+
+var o = {
+    set a(value) {
+        this.val = value;
+    }
+};
+
+var o = {d: 1};
+Object.defineProperty(o, 'c', {
+    set: function(value) {
+        this.val = value;
+    }
+});
+/*eslint accessor-pairs: "error"*/
+
+var o = {
+    set a(value) {
+        this.val = value;
+    },
+    get a() {
+        return this.val;
+    }
+};
+
+var o = {d: 1};
+Object.defineProperty(o, 'c', {
+    set: function(value) {
+        this.val = value;
+    },
+    get: function() {
+        return this.val;
+    }
+});
+
+/*eslint accessor-pairs: ["error", { "getWithoutSet": true }]*/
+
+var o = {
+    set a(value) {
+        this.val = value;
+    }
+};
+
+var o = {
```

Comment:
__[development][eslint]__
* Error - 'o' is already defined (no-redeclare)
* Error - Setter is not present (accessor-pairs)